backend/src/schema/resolvers/groups.js

```diff
@@ -97,7 +97,7 @@ export default {
         if (isMember) {
           cypher = `MATCH (user:User)-[membership:MEMBER_OF]->(group:Group)
                     WHERE user.id = $userId
-                    AND membership.role IN ['usual', 'admin', 'owner']
+                    AND membership.role IN ['usual', 'admin', 'owner', 'pending']
```

Comment:
is there no way to reduce the two queries to one, so this problem will never araise in the future anymore?